rules.neon

```diff
@@ -16,6 +16,7 @@ parameters:
 		illegalConstructorMethodCall: %featureToggles.bleedingEdge%
 	strictRules:
 		allRules: true
+		disallowedLooseComparsion: [%strictRules.allRules%, %featureToggles.bleedingEdge%]
```

Comment:
Typo

Reply:
OMG I was writing this word wrong my whole life :-D Fixed